src/components/form/dt-tags/dt-tags.js

```diff
@@ -126,6 +129,29 @@ export class DtTags extends DtMultiSelect {
     `;
   }
 
+  _validateRequired() {
+    const { value } = this;
+
+    var empty = true;
+
+    if (value != undefined) {
+      for (var i = 0; i < value.length; i++) {
+        if (value[i].charAt(0) != '-') {
```

Comment:
This could cause errors if the array has null values (e.g. `[ 'first', null ]`). So to handle that, you can change this to `if (!value[i] || value[i].charAt(0) !== '-') {`

Reply:
Actually, there are other errors in another component that inherits this one where the value is not a string array but an array of objects, so we should update this a bit:

```
empty = value.every((i) =>
        !i ||
        (typeof i === 'string' && i.charAt(0) !== '-')
      )
```

The first line makes sure each value in the array isn't null/undefined/empty.

The second line which check if the first character is a hyphen only if the value is a string. The user-connection component inherits this and will need to be updated eventually, but it has its value as an array of objects. So it's unit tests are failing right now, but this should fix it.